tools/dgeni/index.bzl

```diff
@@ -26,28 +27,42 @@ def _dgeni_api_docs(ctx):
 
   # Pass the path to the output directory. This will be used to instruct Dgeni where the docs
   # output should be written to. (e.g. bazel-out/bin/src/docs-content)
-  args.add(output_directory.path)
+  args.add(output_dir_path)
 
   # Pass each specified entry point and it's corresponding entry points to Dgeni. This will then
   # be used to resolve the files that need to be parsed by Dgeni.
   for package_name, entry_points in ctx.attr.entry_points.items():
     args.add(package_name)
     args.add_joined(entry_points, join_with = ",")
 
+    for entry_point in entry_points:
+      expected_outputs += [
+        # Declare the output for the current entry-point. The output file will always follow the
+        # same format: "{output_folder}/{package_name}-{entry_point_name}.html"
+        # (e.g. "api-docs/material-slider.html")
+        ctx.actions.declare_file("%s/%s-%s.html" % (output_dir_name, package_name, entry_point))
+      ]
+
+    # Small workaround that ensures that the "ripple" API doc is properly exposed as an output
+    # of the packaging rule. Technically Dgeni should not output the "ripple" directory as
+    # it's own entry-point. TODO(devversion): Support sub API docs for entry-points
+    if package_name == "material":
+      expected_outputs += [
+        ctx.actions.declare_file("%s/%s-%s.html" % (output_dir_name, package_name, "ripple"))
```

Comment:
@jelbourn I  didn't bother too much about it (yet), because I want to get this going now. I will follow-up on this later as a low priority.